### DISCUSSION TITLE
Disable Refresh on Return

### DISCUSSION THREAD

**Discussion by deepdelirious** [zanllp/infinite-image-browsing#702] [Ideas]
The recent work to enable refreshing in walk mode is causing problems with my work process. Any chance you could add a config option to disable that?

**Comment by zanllp** [comment]
What issues have the recent changes caused?
 If it's a speed issue, you can try updating to the latest commit.

**Comment by deepdelirious** [comment]
No real issue, just my stubbornness. I like to generate a large batch of images, and then spend time looking at them while the next batch generates. When I switch to a different tab and come back, I lose my place.

**Comment by zanllp** [comment]
I will add a switch in a few days to control this behavior.

Currently, the auto-refresh in walk mode is only enabled when you are browsing the first 128 images.

**Comment by zanllp** [comment]
![image](https://github.com/user-attachments/assets/8eb75cec-acbb-4ed1-8b75-68e553380dd9)
